models/lung_nodule_ct_detection/configs/inference_raw.json

```diff
@@ -0,0 +1,55 @@
+{
+    "data_list_file_path": "$@bundle_root + '/LUNA16_datasplit/original/dataset_fold0.json'",
+    "data_file_base_dir": "/home/canz/Projects/datasets/LUNA16/93176/Images",
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "DeleteItemsd",
```

Comment:
May I know the error or problem if don't remove these 2 keys?

Thanks.

Reply:
There is not error resulted. I have removed this transform. I added only to make sure the inference did not use ground truth.